src/WeightInitializers.jl

```diff
@@ -1,6 +1,7 @@
 module WeightInitializers
 
 import PrecompileTools: @recompile_invalidations
+using PartialFunctions, Random, SpecialFunctions, Statistics, LinearAlgebra
```

Comment:
this line is not needed, right? Just LinearAlgebra needs to be put inside `@recompile_invalidations`

Reply:
ah yes, I missed a couple of details in the rebase apparently. I fixed that and also added the missing inits to the non differentiable list